### PR TITLE
sips: add page

### PR DESCRIPTION
**Pull request by martin-juul**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/osx/sips.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# sips
+
+> Apple Scriptable Image Processing System.
+> Raster/Query images and ColorSync ICC Profiles.
+
+- Specify an output directory so originals does not get modified:
```
Comment (agnivade) [inline pages/osx/sips.md]: so _that_ originals _do_ not get modified.

File: pages/osx/sips.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`sips --out /path/to/out ...`
```
Comment (agnivade) [inline pages/osx/sips.md]: path/to/out should be in tokens `{{}}`. Also, what are the `...` for ?

File: pages/osx/sips.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Resample image at specified size, Image aspect ratio may be altered:
```
Comment (agnivade) [inline pages/osx/sips.md]: Should it be resample or resize ?

Comment (martin-juul) [reply pages/osx/sips.md]: this [page](http://www.richmediacs.com/user_manuals/RMCS_PS_Training/ResizeVsResample.html) explains the difference between resizing and resampling well.

The tldr, is when resizing you're redistributing the amount of pixels. Compared to when you're resampling you're adding or subtracting the amount of pixels in the image.

File: pages/osx/sips.md (hunk 2 above)
Comment (martin-juul) [reply pages/osx/sips.md]: I will fix that, the dots were meant to imply the rest of the users command (appending to another command). Should i change it to {{rest_of_command}} or just remove it?

File: pages/osx/sips.md (hunk 1 above)
Comment (martin-juul) [reply pages/osx/sips.md]: Sorry, i'll fix that.

**Pull request merged**